docs/graphql/manual/remote-schemas/schema-auth.rst

```diff
@@ -0,0 +1,103 @@
+.. meta::
```

Comment:
:+1: for splitting the index into two files: adding remote schema and auth in remote schema